test/runtests.jl

```diff
@@ -1,62 +1,64 @@
+Pkg.checkout("StaticArrays", "sd/fixedsizearrays")
```

Comment:
the question remains:  what's special about this branch of `StaticArrays`?  can we get by without it for `FreeTypeAbstraction`?  if so, then we could still support julia 0.5 if i understand things correctly.